crates/swift-bridge-cli/src/clap_app.rs

```diff
@@ -101,3 +102,34 @@ fn create_package_command() -> Command<'static> {
                 .help("The name for the Swift Package"),
         )
 }
+
+fn create_bridges_command() -> Command<'static>
+{
+    Command::new("parse-bridges")
+        .about("Parse bridge library files and output generated headers")
+        .after_help(
+            "By default, the output it written at 'generated' folder at the process working directory. \
+                  Use -o/--output to override the output location"
+        )
+        .arg(
+            Arg::new("crate-name")
+                .help("Crate name for which the bridging headers are generated; \
+                          used as a part of header names")
+                .action(ArgAction::Set)
+                .required(true)
+        )
+        .arg(
+            Arg::new("source-files")
+                .help("source file(s) containing #[swift_bridge::bridge] macro")
+                .multiple_values(true)
+                .required(true)
+        )
+        .arg(
+            Arg::new("output")
+                .long("output")
+                .short('o')
+                .takes_value(true)
+                .value_name("PATH")
+                .help("Output destination folder; subfolders are created automatically")
```

Comment:
No need to mention that subfolders are created automatically.
This information isn't useful to the user.

Reply:
I found this helpful :) 

But I might be in the minority here, so I removed this.